docs/config.rst

```diff
@@ -572,6 +572,18 @@ Features
    .. _RVM: https://rvm.io/
    .. _RBENV: https://github.com/rbenv/rbenv
 
+.. attribute:: LP_ENABLE_TERRAFORM
+   :type: bool
+   :value: 0
+
+   Display the currently activated `Terraform` workspace.
```

Comment:
This still needs an underline:
```suggestion
   Display the currently activated `Terraform`_ workspace.
```
You can still see in the built docs that it isn't a link: https://liquidprompt--669.org.readthedocs.build/en/669/config.html#LP_ENABLE_TERRAFORM